<launch>
	<arg name="tf_prefix" value="/skeleton" />

	<node pkg="openni_tracker" name="openni_tracker" type="openni_tracker" output="screen">
		<param name="tf_prefix" value="$(arg tf_prefix)" />
		<param name="fixed_frame" value="openni_depth_frame" />
	</node>


	<node pkg="test_detections" name="hand_detection" type="hand_detection" output="screen">
		<rosparam file="$(find test_detections)/params/marker_params.yaml" command="load" />
		<param name="tf_prefix" value="$(arg tf_prefix)" />
		<param name="fixed_frame" value="openni_depth_frame" />
	</node>
</launch>
